SU-SWS/stanford_news
# D8CORE-2832: fix look of focus around news linked <h2>.

# READY FOR REVIEW


# Summary
Fixes how the look of the focus looks on <h2>

# Needed By (Date)
Soon

# Urgency
I believe https://github.com/SU-SWS/saa_victoria_subtheme/pull/139 would benefit.
I had been waiting for https://github.com/SU-SWS/decanter/pull/785, but there's really no need.

# Steps to Test

1. Check out this branch
2. Clear the cache
3. Verify the focus looks like:
![Screen Shot 2021-02-04 at 1 14 27 PM](https://user-images.githubusercontent.com/284440/106956660-c5fd6d80-66eb-11eb-9dd4-f5de947891a5.png)



# Affected Projects or Products
- D8Core, SoE, SAA

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2832
- Other PRs
   https://github.com/SU-SWS/decanter/pull/785
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@ishahism 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
